hcipy/aperture/realistic.py

```diff
@@ -750,3 +750,210 @@ def segment_with_spider(segment):
 		return elt_aperture_with_spiders, elt_segments
 	else:
 		return elt_aperture_with_spiders
+
+def make_gmt_aperture(normalized=False, with_spiders=True, return_segments=False):
+	'''Make the Giant Magellan Telescope aperture.
+
+	The primary mirror parameters come from the GMT Observatory Architecture Documentation (GMT-REQ-03215, Rev. F):
+	https://www.gmto.org/resources/slpdr/ . Small corrections have been applied to match to the actual pupil from internal GMTO files.
+
+	Parameters
+	----------
+	normalized : boolean
+		If this is True, the outer diameter will be scaled to 1. Otherwise, the
+		diameter of the pupil will be 25.448 meters.
+	with_spiders : boolean
+		If this is False, the spiders will be left out. Default: True.
+	return_segments : boolean
+		If this is True, the segments will also be returned as a list of Field generators.
+
+	Returns
+	-------
+	Field generator
+		The GMT aperture.
+	elt_segments : list of Field generators
+		The segments. Only returned when `return_segments` is True.
+	'''
+	
+	##
```

Comment:
Only single `#` for comments. Also no empty comments. Same for below.

Reply:
Done